CmdStarter.Lib/Reflection/Loader.cs

```diff
@@ -81,5 +83,34 @@ internal static void LoadAliases(ICustomAttributeProvider provider, IdentifierSy
                 receptacle.AddAlias(alias);
             }
         }
+
+        internal static void LoadAutoCompletes(ICustomAttributeProvider provider, Option option)
+        {
+            var completionDelegates = GetAutoCompleteAttributes(provider)
+                .Select(attribute => attribute.Context);
+
+            foreach (var completion in completionDelegates)
+            {
+                option.AddCompletions(completion);
+            }
+        }
+
+        internal static void LoadAutoCompletes(ICustomAttributeProvider provider, Argument argument)
+        {
+            var completionDelegates = GetAutoCompleteAttributes(provider)
+                .Select(attribute => attribute.Context);
+
+            foreach (var completion in completionDelegates)
+            {
+                argument.AddCompletions(completion);
+            }
+        }
+
+        internal static IEnumerable<AutoCompleteAttribute> GetAutoCompleteAttributes(ICustomAttributeProvider provider)
```

Comment:
As it is only used by this class, couldn't it be `private`?

Reply:
Changed.